docs/application/dotnet/guides/nui/styling-controls-with-xaml.md

```diff
@@ -0,0 +1,200 @@
+# Styling UI components with XAML
+Styling of components in NUI is possible in two ways:
+  - `ViewStyle`: designed for simple themes.
+  -  XAML file: designed for advanced themes. With XAML file, styles change in runtime and can be shared between the applications.
+
+## ViewStyle theme
+The basic styling of NUI components is based on the `ViewStyle` class and its derivatives. `ViewStyle` defines attributes and stores their values. 
+
+The following is a list of the most common attributes:  
+- name
+- state 
+- subState
+- position
+- scale
+- orientation
+- padding 
+
+You can find all the attributes and their details on [ViewStyle attributes](https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyle.cs) page.
+
+In addition, styles can also be defined for the components with following states:
+| State              | Description                                                                                                                                          |
+|--------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `Normal`           | Specifies the default style.                                                                                                                         |
+| `Focused`          | Specifies the state where all the keystrokes and events are recieved by the widget. There can only be one widget in the active window at a time in this state. |
+| `Disabled`         | Specifies the state where the widget is disabled. All other events are ignored in this state.                                                          |
+| `Selected`         | Specifies the state where a widget is selected.                                                                                                      |
+| `Pressed`          |  Specifies the state where the widget receives touch event.                                                                                                                                                   |
+| `DisabledFocused`  | Specifies the state that is similar to the `Focused` state, but there is no possibility to change the widget state.                                                                        |
+| `SelectedFocused`  | Specifies the state where the widget is checked and all events are received.                                                                                            |
+| `DisabledSelected` | Specifies the combination of disabled and selected states.                                                                                                       |
+The NUI also defines the `ViewStyle` derived classes for components such as `ImageView`, `TextLabel` and `TextField`:
```

Comment:
I think there is some issue, why is line 31 coming in the table when rendered!

![image](https://user-images.githubusercontent.com/43839737/106561072-1048e800-654e-11eb-9f63-7075994fb051.png)


Please verify

Reply:
I think giving one line space will do.